topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -0,0 +1,629 @@
+> > 3. Both the CLIP experiment and the control are closely related. It is maybe wise to check the correlation of the two samples.
+> {: .solution}
+>
+{: .question}
+
+For additional information on how to interpret **plotFingerprint** plots, read the information [here](https://deeptools.readthedocs.io/en/latest/content/tools/plotFingerprint.html#background).
+
+## Correlation between Samples
+
+We have seen that our input control and our CLIP experiment might be strongly correlated, meaning, that our potential RBFOX2 binding regions are not truly enrichment when comparing to a our control.
+We are therefore going to further check the correlation between our control and CLIP experiment.  
+
+> ### {% icon hands_on %} Hands-on: Assessing correlation between samples
+>
+> 1. **multiBamSummary** {% icon tool %} with the following parameters:
+>    - *"Sample order matters"*: `No`
+>    - *"Bam file"*: `both bam files` (output of **UMI-tools deduplicate** {% icon tool %})
+>    - *"Choose computation mode"*: `Bins`
+>     - *"Bin size in bp"*: `1000`
+>    - *"Show advanced options"*: `no`
+> 2. **plotCorrelation** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Matrix file from the multiBamSummary tool"*: `outFile` (output of **multiBamSummary** {% icon tool %})
+>    - *"Correlation method"*: `Spearman`
+>    - *"Plotting type"*: `Heatmap`
+>    - *"Skip zeros"*: `Yes`
+>    - *"Image file format"*: `png`
+> 3. View the output image.
+> ![correlation](../../images/clipseq_correlation.png "Heatmap of correlation matrix generated by <b>plotCorrelation</b>.")
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. Why do we want to skip zeros in plotCorrelation?
+> 2. What happens if the Spearman correlation method is replaced by the Pearson method?
+> 3. What does the correlation tell you about our data?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Large areas of zeros would lead to a correlation of these areas. The information we would get out of this computation would be meaningless.
+> > 2. The clusters would be different.
+> > 3. Despite the result of `plotFingerprint`, we see that the correlation between our experiment and control is not nearly as significant as we thought.
+> {: .solution}
+>
+{: .question}
+
+# Step 7: Peakcalling
+
+Peakcalling is one of the most important steps in the data analysis of CLIP-Seq, next to mapping. There exist a variety of peakcallers, each with different underlying assumptions and parameters. It is up to you to find the best suited method and set of parameters, that is why, it is difficult (nearly impossible) to find a generic solution for every data set. We are going to use `PEAKachu` to find possible binding motifs for our data. `PEAKachu` is able to incorporate control data in contrast to other peakcallers like `Piranha`, thus allowing to find binding regions that are significantly enriched in comparison to our control (input) data.
```

Comment:
I know what you want to say, but `it is up to you ` does not help our audience. Maybe remove this sentence and explaining that there are many different peakcallers, just name a few and then explain why this one is great for this dataset.